track/templates/includes/fr/modal.html

```diff
@@ -5,40 +5,59 @@
 	  <div id="modal-content" tabindex="-1" role="dialog" aria-labelledby="table compliance definitions" class="relative bg-white w-3/4 m-auto flex-col py-2 overflow-auto flex-initial">
 	  	<div id="close" class="w-full text-right pr-6"><span class="close-symbol"><a id="close-btn" class="no-underline text-black text-3xl lg:text-5xl" href="#">&times;</a></span></div>
 	  	<div class="container lg:w-5/6 mx-auto">
-		    <h2 id="modal-header" tabindex="-1" class="text-3xl lg:text-5xl font-bold mb-4">Compliance definitions</h2>
-		    <p class="text-xl lg:text-2xl mb-8">We are tracking Canadian government web security compliance based off the following criteria:</p>
-		    <h3 class="text-xl lg:text-2xl font-bold mb-4">IT PIN</h3>
-		    <p class="text-base lg:text-lg">Lorem ipsum dolor sit amet, consectetur adipiscing elit. Nulla suscipit dui nec consequat bibendum. Nam condimentum ac turpis non vestibulum.</p>
-		    <p class="text-base lg:text-lg">Vestibulum ante ipsum primis in faucibus orci luctus et ultrices posuere cubilia Curae; Aliquam id lacus ac erat congue placerat.</p>
-		    <p class="text-base lg:text-lg"><a href="" class="text-https-blue hover:text-https-blue-darker font-bold">Learn more</a></p>
+		    <h2 id="modal-header" tabindex="-1" class="text-3xl lg:text-5xl font-bold mb-4">Critères de conformité</h2>
+		    <p class="text-xl">Les colonnes du tableau font état de la conformité aux critères énumérés dans les directives de la Politique.</p>
+		    <p class="text-xl mb-8">Chaque colonne indique le pourcentage global de conformité de chaque organisation ou domaine. Cliquez sur «&nbsp;Afficher les détails&nbsp;» ou «&nbsp;Afficher les sous-domaines&nbsp;» pour afficher les valeurs des colonnes pour des domaines spécifiques.</p>
+		    <h3 class="text-xl lg:text-2xl font-bold mb-4">Conforme à l’AMPTI</h3>
+		    <p class="text-base lg:text-lg">Indique si un domaine est conforme à l’ensemble des directives de l’AMPTI 2018-01.</p>
+		    <p class="text-base lg:text-lg underline">Valeurs</p>
+		    <ul class="list-reset text-base lg:text-lg">
+		    	<li class="mb-2">Non&nbsp;: le domaine n’est pas conforme à l’AMPTI 2018-01.</li>
+		    	<li>Oui&nbsp;: le domaine est conforme à l’AMPTI 2018-01.</li>
+		    </ul>
 
-			<hr class="border mt-4 mb-4">
+			<hr class="border mt-6 mb-6">
 
-			<h3 class="text-xl lg:text-2xl font-bold mb-4">Enforces HTTPS</h3>
-		    <p class="text-base lg:text-lg">Lorem ipsum dolor sit amet, consectetur adipiscing elit. Nulla suscipit dui nec consequat bibendum. Nam condimentum ac turpis non vestibulum.</p>
-		    <p class="text-base lg:text-lg">Vestibulum ante ipsum primis in faucibus orci luctus et ultrices posuere cubilia Curae; Aliquam id lacus ac erat congue placerat.</p>
-		    <p class="text-base lg:text-lg"><a href="" class="text-https-blue hover:text-https-blue-darker font-bold">Learn more</a></p>
+			<h3 class="text-xl lg:text-2xl font-bold mb-4">Utilise le protocole HTTPS</h3>
```

Comment:
The table says "applique le protocole HTTPS" - should be consistent one way or another